src/components/SIngleArticle.jsx

```diff
@@ -81,6 +85,28 @@ function SingleArticle() {
     }
   }, [isPosted]);
 
+  const deleteComment = (id) => {
+    setDeleting(true);
+    deleteCommentById(id)
+      .then((response) => {
+        setDeleting(false);
+        setIsDeleted(true);
+        setComments(comments.filter((comment) => comment.comment_id !== id))
+      })
+      .catch((error) => {
```

Comment:
having a .catch here means you don't need it inside the deleteCommentById function, same for the api functions